app/controllers/favorites_controller.rb

```diff
@@ -2,13 +2,13 @@ class FavoritesController < ApplicationController
   before_action :require_signin
   before_action :set_movie
   def create
-    @movie.fans << current_user
+    @movie.fans << Current.user
```

Comment:
You can make this code more readable by flipping the messages for example:

`Current.user.favourite_moives << @movie`
 or
`Current.user.is_fan_of @movie`

Reads nicer, behaves as a self documenting code :)